### ISSUE TITLE
Create oscillator wavetable with MSEG-like UI

### ISSUE THREAD

**Issue by porzione** [surge-synthesizer/surge#7039]
Is it possible to add another oscillator with a hand-drawn curve, with an interface similar to the MSEG editor?

**Comment by Andreya-Autumn** [comment]
Not impossible at all! Also not easy maybe. We've considered MSEG-ish interfaces for the waveshaper also, so why not consider this. I can almost certainly guarantee that it won't be until XT2 though, so will put it in that milestone. Thanks for the request!

**Comment by mkruselj** [comment]
Basically the same thing as #6783.

**Comment by baconpaul** [comment]
So for the wave shaper, aliasing is a real concern.

For the oscillator, I think the request is really "can I use a UI like the mseg editor to make a wavetable for the wavetable oscillator". Anything else would sound like garbage. We have code (not turned on) to do that with the lua formula editor so its entirely possible; just a lot of UI work.

**Comment by porzione** [comment]
I was thinking about editing just one waveform but if I can edit the wavetable it would be fantastic.

**Comment by baconpaul** [comment]
Yeah my point was once you edit a waveform the only way to make audio from it is to make it a wavetable anyway! We don’t have a performant Alias suppressed general shape oscillator other than wavetable basicalky

**Comment by mkruselj** [comment]
This feature would also majorly benefit from #6469!
